db/migrate/20181108032049_remove_unused_indexes.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveUnusedIndexes < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :coins, column: :volume24 # name: "index_coins_on_volume24"
```

Comment:
Why not pass the name in as a parameter?

Reply:
Great question - passing in the `column` allows the migration to be rolled back automatically, otherwise the migration is one way only.